pom.xml

```diff
@@ -108,7 +108,7 @@
         <!-- keep in-sync with dataformat-yaml -->
         <json-path.version>4.1.1</json-path.version>
         <zkclient.version>0.11</zkclient.version>
-        <scala-library.version>2.12.12</scala-library.version>
+        <scala-library.version>2.13.5</scala-library.version>
```

Comment:
Any reason not to use the latest `2.13.6`?

Reply:
It probably did not exist when I opened the PR for the first time.